### DISCUSSION TITLE
Why am I viewing the source code mode instead of reading mode in the backlink How can I modify this point?

### DISCUSSION THREAD

**Discussion by xiyuweifeng267** [RyotaUshio/obsidian-pdf-plus#194] [Q&A]
<img width="542" alt="Snipaste_2024-05-07_21-54-28" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/54426905/00f134e1-604d-4bce-97db-70fd821fd03c">
<img width="1046" alt="Snipaste_2024-05-07_21-53-45" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/54426905/1b340830-6328-470e-83ea-8598e1069af4">

**Comment by RyotaUshio** [comment]
It's another plugin called Better Search Views. It's mentioned in the README.
